### DISCUSSION TITLE
Test-driving Inspector

### DISCUSSION THREAD

**Discussion by crazyideas21** [noise-lab/iot-inspector-client#169] [Q&A]
If you're test-driving Inspector, ask your questions here!

**Comment by patrickschmahl** [comment]
Hi, I got an error while installing the software; Failed to install Python driver.

**Reply by crazyideas21** [reply]
Could you upload the `init.log` file? See https://github.com/nyu-mlab/iot-inspector-client/issues/187#issuecomment-1378982352

**Reply by patrickschmahl** [reply]
![Screen Shot 2023-01-19 at 1 53 15 PM](https://user-images.githubusercontent.com/122311086/213534516-45ee9d70-ab67-43bf-abff-4334da98111f.png)
I can't get past this step. I download the .zip file, then click on it, then double click on the Inspector icon, then put in my computer password, then this happens.
